requirements.txt

```diff
@@ -1,7 +1,11 @@
 gym>=0.18.0
 pygame==2.0.0
 numpy>=1.18.0
+<<<<<<< HEAD
```

Comment:
The reason tests are failing is because the git merge conflict guards got inserted into in the code.

Reply:
The magical errors are the first test, not this. That wouldn't have fixed it, but hopefully PyMunk 5.7.1 will prevent me from having to fix i.t